packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.vue

```diff
@@ -44,6 +44,7 @@ const isTouched = ref(false);
         v-else
         v-model="isChecked"
         :aria-label="props.hideLabel ? props.label : undefined"
+        :title="props.hideLabel ? props.label : undefined"
```

Comment:
I'd suggest to enhance the description text inside the jsdoc of the `label` property to inform the devs that this will happen, so they understand that all users will be able to see the text even when hidden (and are not tempted to take a short-cut with an " " string e.g. ^^)

Reply:
done